src/main/java/green/citibike/CitiController.java

```diff
@@ -87,17 +85,17 @@ private double haversine(double lat1, double lon1, double lat2, double lon2) {
     }
 
     private double haversine(String station1, String station2) {
-        double lat1 = stationStatusMap.get(station1).lat;
-        double lat2 = stationStatusMap.get(station2).lat;
-        double lon1 = stationStatusMap.get(station1).lon;
-        double lon2 = stationStatusMap.get(station2).lon;
+        double lat1 = stationStatusMap.get(station1).getLat();
+        double lat2 = stationStatusMap.get(station2).getLat();
+        double lon1 = stationStatusMap.get(station1).getLon();
+        double lon2 = stationStatusMap.get(station2).getLon();
         return haversine(lat1, lon1, lat2, lon2);
     }
 
     private StationStatus findClosestStationId(double lat, double lon, boolean hasBike) {
         String stationId = findClosestStationCoords(lat, lon);
 
-        if (stationStatusMap.get(stationId).num_bikes_available == 0) {
+        if (stationStatusMap.get(stationId).getNumBikesAvailable() == 0) {
```

Comment:
Do you actually mean `numBikesAvailable == 0`?

Reply:
Yes -- so it doesn't bother searching unless this station has no bikes available

Reply:
Why would that matter?